packages/terra-avatar/src/common/AvatarUtils.jsx

```diff
@@ -71,7 +71,7 @@ const generateImagePlaceholder = (alt, isAriaHidden, variant) => {
  */
 const generateImage = (image, alt, isAriaHidden, variant, handleFallback) => {
   const icon = generateImagePlaceholder(alt, isAriaHidden, variant);
-  return <TerraImage className={cx('image')} src={image} placeholder={icon} alt={alt} onError={handleFallback} />;
+  return <TerraImage className={cx('image')} src={image} placeholder={icon} alt={alt} onError={handleFallback} Fit={FitTypes.COVER} />;
```

Comment:
You could drop the additional import and just use the string.

```suggestion
  return <TerraImage className={cx('image')} src={image} placeholder={icon} alt={alt} onError={handleFallback} Fit={'cover'} />;
```

Reply:
Fixed here https://github.com/cerner/terra-core/pull/2494/commits/be42fad09a1ef9b1e6caf2b5042500e0e494d4db.